app/javascript/chat/channelDetails.jsx

```diff
@@ -114,21 +114,47 @@ class ChannelDetails extends Component {
     let pendingInvites = [];
     if (channel.channel_mod_ids.includes(window.currentUser.id)) {
       // eslint-disable-next-line
-      searchedUsers = this.state.searchedUsers.map(user => (
-        <div className="channeldetails__searchedusers">
-          <a href={user.path} target="_blank" rel="noopener noreferrer">
-            {user.title}
-          </a>
-          {' '}
+      searchedUsers = this.state.searchedUsers.map(user => {
+        let invite = (
           <button
             type="button"
             onClick={this.triggerInvite}
             data-content={user.id}
           >
             Invite
           </button>
-        </div>
-      ));
+        );
+        if (channel.channel_users.includes(user)) {
```

Comment:
Hmmm I’m not sure they’d be the equivalent objects. So `user === otherUser` may not be work, but `user.username === otherUser.username` should work correctly. They may be. I’m just not sure.

I say I'm not sure because I'm not sure. This _may_ work. Not totally sure of the whole context here.

Reply:
Compares via user ID instead of whole user object now.